config/default.js

```diff
@@ -15,67 +15,6 @@ module.exports = {
   defaultMimeType: process.env.DEFAULT_MIME_TYPE || 'application/ld+json',
   indexType: process.env.INDEX_TYPE || 'sinopia',
   indexUrl: process.env.INDEX_URL || 'http://localhost:9200',
-  // Note that InputLookupSinopia expects uri and label fields.
-  indexMappings: process.env.INDEX_FIELD_MAPPINGS
```

Comment:
Looks like there's some code in `search.js` that may still use the old name of this config value (`indexFieldMappings`). Could possibly be done in a separate commit or PR.

Reply:
I removed search.js. Given how easy it is to perform a query with curl this seems unnecessary and more likely to confuse, as it does not accurately reflect the queries that are performed by the editor.

Reply:
:clinking_glasses: